priv/templates/phx.gen.live/show.html.heex

```diff
@@ -14,16 +14,14 @@
 
 <.back navigate={~p"<%= schema.route_prefix %>"}>Back to <%= schema.plural %></.back>
 
-<%%= if @live_action in [:edit] do %>
-  <.modal id="<%= schema.singular %>-modal" show on_cancel={JS.patch(~p"<%= schema.route_prefix %>/#{@<%= schema.singular %>}")}>
-    <.live_component
-      module={<%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Live.FormComponent}
-      id={@<%= schema.singular %>.id}
-      title={@page_title}
-      action={@live_action}
-      <%= schema.singular %>={@<%= schema.singular %>}
-      navigate={~p"<%= schema.route_prefix %>/#{@<%= schema.singular %>}"}
-    />
-  </.modal>
-<%% end %>
+<.modal :if={if @live_action in [:edit]} id="<%= schema.singular %>-modal" show on_cancel={JS.patch(~p"<%= schema.route_prefix %>/#{@<%= schema.singular %>}")}>
```

Comment:
```suggestion
<.modal :if={@live_action == :edit} id="<%= schema.singular %>-modal" show on_cancel={JS.patch(~p"<%= schema.route_prefix %>/#{@<%= schema.singular %>}")}>
```
Removing the leftover `if` and changed to use == instead of comparing with a list with one element.